### ISSUE TITLE
[Feature Request] postman intial and current value

### ISSUE THREAD

**Issue by bugada** [usebruno/bruno#1216]
Hello,

thanks for your awesome work.
The variable in the environment or pre-request are saved on the git repo.
So if I'm testing something and one of my colleague changes the same variable it's a problem.

Could you implement an "initial value" saved on the collection, and a "current value" not saved, so if one changes the current it won't be saved in the git repo? You can also call it "local value". Obviously the local value, if present, must overwrite the initial value 

Thanks
Andrea

**Comment by MelGrubb** [comment]
This sounds related to #1033 
Any mechanism that would let me set a variable value, but NOT save that value to the file would be great.

**Comment by bugada** [comment]
Yep the 1033 Is an awesome request, but if you have 10 variables and usually you have to change 1 or 2 values a default value would be great. Just save those default values on a file local.json like secret.json and let the user to customize it: for example when the popup is showed prefill with default values, a button "save as default" will alter then  and another "continue" will use those values, even changed, without saving it.

**Comment by MelGrubb** [comment]
> ...Just save those default values on a file local.json like secret.json and let the user to customize it...

That's exactly what I had in mind, although I think I'd want these values to be even more transient, so that they don't carry over to the next session, or maybe even if I closed the tab and reopened it. I want an in-memory-only option so that I don't accidentally run one of these requests with the same values in the future. Maybe the dialog could have checkboxes to determine whether or not the values would be retained at all.

**Comment by chimit** [comment]
Probably, we can consider these ephemeral variables as [secrets](https://docs.usebruno.com/secrets-management/overview). It will prevent these values from being pushed to git.

**Comment by the-corp-mark** [comment]
Any progress on this?

**Comment by pindab0ter** [comment]
> Another idea how to support default values is via `.env.default` which should be under version control and be overridden by your local `.env` if specified.

This would definitely also work. Is there any news on this issue?

**Comment by GoncaloBastos** [comment]
Any update on this one? 

It would be a big QoL improvement when multiple people are working on the same repo. I know using secrets sort of achieves this but it loses a 'default' value.
